theia-plugin/src/extension.ts

```diff
@@ -1,21 +1,174 @@
+// tslint:disable:no-var-requires
+// tslint:disable:no-require-imports
+// tslint:disable:no-use-before-declare
 
 /**
  * Generated using theia-plugin-generator
  */
 
 import * as theia from '@theia/plugin';
+const tooling = require('jsonforms-tooling');
```

Comment:
can't we use an import instead of require here?

Reply:
I replaced the `require` with `import` although it seems, that we have the  `Cannot find module 'jsonforms-tooling'` error again:
![screenshot_2018 12 16_21h11m07s_010_](https://user-images.githubusercontent.com/1368405/50058341-28dfc380-0177-11e9-98c8-7517db37680e.png)

Maybe you could try the latest commit (run a `npm install` and a `npm run compile` inside the root folder) and tell me if you have the same issue or if it works fine for you